appeals/web/src/server/appeals/appeal-details/assign-user/__tests__/assign-user.test.js

```diff
@@ -19,12 +18,6 @@ describe('assign-user', () => {
 		usersService.getUsersByRole = jest.fn().mockResolvedValue(activeDirectoryUsersData);
 		// @ts-ignore
 		usersService.getUserByRoleAndId = jest.fn().mockResolvedValue(activeDirectoryUsersData[0]);
-
-		nock('https://graph.microsoft.com/')
-			.get(
-				`/v1.0/groups/${config.referenceData.appeals.caseOfficerGroupId}/members?$select=id,displayName,userPrincipalName`
-			)
-			.reply(200, activeDirectoryUsersData);
```

Comment:
I was going to update this, but running the tests first I noticed it made no difference; because of the mocked service functions above, we don't hit the API.